dev/reference/configuration/tidb-server/tidb-specific-variables.md

```diff
@@ -381,3 +396,123 @@ Usage example:
+
+    ```sql
+    select * from t, (select aa from t1 group by aa) tmp_t where t.a = tmp_t.aa
+    ```
+
+    If `t1` is limited to be unique and not null in the `aa` column. You can use the following statement, without aggregation.
+
+    ```sql
+    select * from t, t1 where t.a=t1.a
+    ```
+
+### tidb_opt_correlation_threshold
+
+- Scope: SESSION | GLOBAL
+- Default value: 0.9
+- This variable is used to set the threshold value of estimating the row count of the current column by a method that estimates the number of rows of the `handle` column using the column histogram. If the order correlation between the current column and the `handle` column exceeds the threshold value, this method is enabled.
+
+### tidb_opt_correlation_exp_factor
+
+- Scope: SESSION | GLOBAL
+- Default value: 1
+- When the method that estimates the number of rows of the `handle` column using the column histogram cannot be used, the heuristic estimation method is used. This variable is used to control the behavior of the heuristic method. 
+    - When the value is 0, the heuristic method is not used.
+    - When the value is greater than 0:
+        - A larger value indicates that an index scan will probably be used in the heuristic method.
+        - A smaller value indicates that a table scan will probably be used in the heuristic method.  
+
+### tidb_enable_window_function
+
+- Scope: SESSION | GLOBAL
+- Default value: 1, indicating enabling the window function feature.
+- This variable is used to control whether to enable the support for window functions. Window functions may use reserved or key words. This might cause SQL statements that could be executed normally cannot be parsed after upgrading TiDB. In this case, you can set `tidb_enable_window_function` to `0`.
+
+### tidb_slow_query_file
+
+- Scope: SESSION
+- Default value: ""
+- When querying `INFORMATION_SCHEMA.SLOW_QUERY`, only the slow query log name is parsed set by `slow-query-file` in the configuration file. The slow query log name is "tidb-slow.log" by default. To parse other logs, set the `tidb_slow_query_file` session variable to a specific file path, and then query `INFORMATION_SCHEMA.SLOW_QUERY` to parse the slow query log based on the set file path. For details, see [Identify Slow Queries](/how-to/maintain/identify-slow-queries.md).
+
+### tidb_enable_fast_analyze
+
+- Scope: SESSION | GLOBAL
+- Default value: 0, indicating not enabling the statistics fast `Analyze` feature.
+- This variable is used to set whether to enable the statistics fast `Analyze` feature.
+- If the statistics fast `Analyze` feature is enabled, TiDB randomly samples about 10,000 rows of data as statistics. When the data is distributed unevenly or the data size is small, the statistics accuracy is low. This might lead to an un optimal execution plan, for example, selecting a wrong index. If the execution time of non-fast `Analyze` statement is acceptable, it is recommended to disable the fast `Analyze` feature.
```

Comment:
```suggestion
- If the statistics `Fast Analyze` feature is enabled, TiDB randomly samples about 10,000 rows of data as statistics. When the data is distributed unevenly or the data size is small, the statistics accuracy is low. This might lead to an unoptimal execution plan, for example, selecting a wrong index. If the execution time of the regular `Analyze` statement is acceptable, it is recommended to disable the `Fast Analyze` feature.
```